core/vm/contracts.go

```diff
@@ -1432,6 +1433,9 @@ func (c *verifyDoubleSignEvidence) Run(input []byte) ([]byte, error) {
 	}
 
 	// basic check
+	if header1.Number.Cmp(maxUnit256) == 1 || header2.Number.Cmp(maxUnit256) == 1 {
```

Comment:
or you can compare length of number here, `header1.Number.Cmp(maxUnit256) == 1 || len(header1.Number.Bytes()) >32`

Reply:
Done